src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -853,6 +858,11 @@ public enum NuGetLogCode
         /// </summary>
         NU5131 = 5131,
 
+        /// <summary>
+        /// InvalidUndottedFrameworkWarning
+        /// </summary>
+        NU5132 = 5132,
```

Comment:
This looks like a warning code. Then, second digit should be between 5-9

See also:

https://github.com/NuGet/NuGet.Client/blob/0e5413751caebf168e7d363c88b9c22a3560fe60/src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs#L10-L35

Reply:
I thought we were basically not doing this anymore?

Reply:
We're still doing it for everything but pack. 
Pack is the one place with the messed up codes. 

We'll eventually run out of errors for pack though, but that's years down the line.

Reply:
I had an issue with a warning in the error code range which is very hard to fix once code is released. If you consider not necessary anymore, feel free to dismiss this comment.

https://github.com/NuGet/Home/issues/8554